connection_plugins/machinectl.py

```diff
@@ -574,13 +574,27 @@ def put_file(self, in_path, out_path):
         if not os.path.exists(to_bytes(in_path, errors='strict')):
             raise AnsibleFileNotFound('file or module does not exist: {0}'.format(in_path))
 
+        # Okay, this is definitely not a great idea to do, that's pretty ugly,
+        # but we have no choice... Let me explain
+        # You cannot "copy-to --force" with machinectl. There is a request to
+        # do that, but unaddressed as of today:
+        #   https://github.com/systemd/systemd/issues/9441
+        # So you cannot overwrite an existing file. This is very annoying when
+        # pushing DIRECTORIES... as the same ansible file (with the same name
+        # on the remote target) must be overwritten for each file in the
+        # directory.
+        # Without removing the file first, we get an error: "file exists".
+        returncode, stdout, stderr = self._run_command('shell', args=['/bin/sh', '-c', f'rm -rf "{out_path}"'], machine=self.machine)
+        if returncode != 0:
+            raise AnsibleError('failed to perform cleanup of file {0}:\n{2}\n{3}'.format(out_path, stdout, stderr))
```

Comment:
I think some of the format placeholders are misnumbered; shouldn't it be the following?:
```
raise AnsibleError('failed to perform cleanup of file {0}:\n{1}\n{2}'.format(out_path, stdout, stderr))
```

Reply:
oops, yes indeed